tsconfig.json

```diff
@@ -1,13 +1,21 @@
 {
   "compilerOptions": {
     "outDir": "./dist/",
+    "rootDir": "./",
+    "target": "es5",
     "module": "commonjs",
+    "moduleResolution": "node",
```

Comment:
Ok I've also removed these flags from CompassQL -- not sure why they are there at all.